include/circt/Translation/ExportVerilog.h

```diff
@@ -27,7 +30,37 @@ class ModuleOp;
 
 namespace circt {
 
-/// Export a module containing RTL, and SV dialect code.
+/// Options which control the emission from CIRCT to Verilog.
+struct VerilogEmitterOptions {
```

Comment:
It is ok to start this here, but this struct will grow to contain a bunch of stuff that has nothing to do with verilog printing (it will affect lowering etc).  I think it should be split out to something like `include/circt/Support/LoweringStyleOptions.h`, and the implementation logic to its own .cpp file.  That can be done as a separate patch if you prefer tho.

The design of this class is very nice!